ratis-netty/src/main/java/org/apache/ratis/netty/server/NettyServerStreamRpc.java

```diff
@@ -109,17 +111,22 @@ void close() {
   private final EventLoopGroup bossGroup = new NioEventLoopGroup();
   private final EventLoopGroup workerGroup = new NioEventLoopGroup();
   private final ChannelFuture channelFuture;
+  private final int clientNum;
```

Comment:
I think we can declare clientNum (or rename it to clientPoolSize) as a local variable in the constructor and use proxies.size() elsewhere.  Then, we don't need this field.